lib/round.rb

```diff
@@ -0,0 +1,52 @@
+class Round
+    attr_reader :deck, :turns, :cards, :number_correct
+    
+    def initialize (deck)
+        @deck = deck
+        @cards = cards
```

Comment:
I do not understand this line, `@cards = cards` what is `cards` and where is it coming from? if you put a `pry` in here, `@cards` is `nil` and stays `nil` as there is no code within this class or in any other file that makes it anything else. You also have an `attr_reader` for it but it is never used outside of this class. My guess is that you could probably get rid of it without any issues.